opennms-base-assembly/src/main/filtered/etc/events/opennms.remote.poller.events.xml

```diff
@@ -1,59 +0,0 @@
-<events xmlns="http://xmlns.opennms.org/xsd/eventconf">
```

Comment:
Same here. Should we keep separate events for remote outages?

Reply:
Then we will recreate it...